settings.gradle

```diff
@@ -1,4 +1,4 @@
-include ':app', ':image',
+include ':app', ':image', ':search',
```

Comment:
We should probably move these onto separate lines....not that there's much call to edit the file in the first place.

Reply:
Fixed it. All the modules are in separate lines now.